src/controllers/relationship.js

```diff
@@ -46,12 +46,12 @@ module.exports = [
             throw error;
```

Comment:
req.auth.id.toString() === friendId.toString()

Isnt this a different error from USER_DOES_NOT_EXIST?

Reply:
It kind of is, but I tried on discord and they seem to pretty much treat it as a not found. That is why I did it here